lib/zendesk_apps_tools/api_connection.rb

```diff
@@ -4,7 +4,7 @@ module APIConnection
     # taken from zendesk/lib/vars.rb
     SUBDOMAIN_VALIDATION_PATTERN = /^[a-z0-9][a-z0-9\-]{1,}[a-z0-9]$/i
     ZENDESK_URL_VALIDATION_PATTERN = /^(https?):\/\/[a-z0-9]+(([\.]|[\-]{1,2})[a-z0-9]+)*\.([a-z]{2,16}|[0-9]{1,3})((:[0-9]{1,5})?(\/?|\/.*))?$/ix
-    EMAIL_REGEX  = /^([\w\.%\+\-]+)@([\w\-]+\.)+([\w]{2,})$/i
+    EMAIL_REGEX  = /^([\w\.%\+\-]+)@([\w\-]+\.)+([\w]{2,})(\/token:\S+)?$/i
```

Comment:
You should probably drop the requirement for a string after token here

Reply:
As Chris said, for token scenario, the email would be `abc@example.com/token` and the token would be provided in the password field.

Reply:
Hey buddy, you're absolutely right! I am so glad you caught this. Bravo Chris.


For context, IRL Chris has brought to my attention that I might have understood the [documentations](https://developer.zendesk.com/rest_api/docs/support/requests#api-token) wrong. 

When prompted to enter credentials when doing `zat create`, after keying in subdomain, developers should be allowed to enter `full_email/token` as username and API TOKEN as password (Api Token is generated in Admin > Channels > API ).

I tested this out and here is the result.

<img width="811" alt="screen shot 2018-12-13 at 9 25 13 pm" src="https://user-images.githubusercontent.com/17760485/49933106-e328be00-ff1e-11e8-8d38-95e6767e2d17.png">